DigitalLearningSolutions.Data/DataServices/UserDataService.cs

```diff
@@ -399,7 +400,8 @@ public int GetDelegateCountWithAnswerForPrompt(int centreId, int promptNumber)
                         FROM Candidates
                         WHERE CentreID = @centreId AND Answer{promptNumber} IS NOT NULL",
                 new { centreId }
-            ).Count(x => !string.IsNullOrWhiteSpace(x)); ;
+            ).Count(x => !string.IsNullOrWhiteSpace(x));
+            ;

```

Comment:
Is this needed?

Reply:
Looks like something that has been around for a while that only just got picked up, will remove